### ISSUE TITLE
 cmd attribute of task is 

### ISSUE THREAD

**Issue by jkrue** [jkrue/jobproxy#56]
The cmd attribute of an task is currently annotated as required. When running a docker container with default command it isn't necessary to set a cmd here.

**Comment by jkrue** [comment]
A adhoc bugfix  is to remove the "not empty" annotation from Task. This change has only a small effects on the external API. Better, but then with much more API incompatibility is to go back to the Task definition:
- a extra cmd attr for the container (which is optional)
- choice between cmd and container (one of both is mandatory)

I integrate the adHoc solution in beta.1.release

**Comment by pbelmann** [comment]
Sorry for the issue and thanks for the fix. Maybe we should schedule the fix that brings API incompatibility for beta.2.release.
@jkrue  if you agree please update the label.

**Comment by jkrue** [comment]
@pbelmann  You're right  ... we should integrate the API incompatible fix in beta.2.release.
